src/pages/AdminDashboard/AdminDashboard.tsx

```diff
@@ -10,6 +10,10 @@ import styles from './AdminDashboard.module.scss';
 import { getAdminTileStats } from '@/graphql/queries';
 import Spinner from '@/components/Spinner/Spinner';
 import ErrorBanner from '@/components/ErrorBanner/ErrorBanner';
+import crown from '@/assets/icons/crown.svg';
```

Comment:
Sorry to be a pain but can you please make the svg files tsx components instead? Have a look at how the other ones in the assets folder are being done